engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -190,9 +194,25 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
   }
 
   // Run action with an outer transaction
-  private def runTransaction[R](action: DBIOAction[R, _ <: NoStream, _ <: Effect]): Future[R] = {
+  private def runTransaction[R](action: DBIOAction[R, _ <: NoStream, _ <: Effect])(implicit ec: ExecutionContext): Future[R] = {
```

Comment:
There are too many `ExecutionContext` flying around. Do you mind making this not-`implicit` and pass it in explicitly where needed?